databroker/eventsource/tests/utils.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+import six
+
+from databroker.eventsource.archiver import ArchiverEventSource
+
+arch_csx_url = 'http://xf23id-ca.cs.nsls2.local:17668'
```

Comment:
@tacaswell Opinions on hard-coding this into databroker?

Reply:
:-1:  It can we replace that with a dummy address?